blaze/expr/datetime.py

```diff
@@ -119,6 +129,27 @@ def utcfromtimestamp(expr):
     return UTCFromTimestamp(expr)
 
 
+class nanosecond(DateTime): _dtype = datashape.int64
```

Comment:
I'm testing what I thought to be the obvious solution: https://github.com/stefanseefeld/blaze/commit/99c1bc637cda7a05c523d75e0f6c4943d450df79, which results in test failures in blaze/server/tests/test_server.py::test_minute_query (https://travis-ci.org/stefanseefeld/blaze/jobs/143455858). Trying to debug this now...